index.ts

```diff
@@ -809,7 +809,8 @@ type U8A = Uint8Array;
 // Steps A, B and C of RFC6979.
 function _abc6979(msgHash: Hex, privateKey: bigint, extraEntropy?: Hex) {
   if (msgHash == null) throw new Error(`sign: expected valid msgHash, not "${msgHash}"`);
-  const num = typeof msgHash === 'string' ? hexToNumber(msgHash) : bytesToNumber(msgHash);
+  let num = typeof msgHash === 'string' ? hexToNumber(msgHash) : bytesToNumber(msgHash);
+  num %= CURVE.n;
```

Comment:
We're using `mod()` function because JS does not have modular division.

Reply:
updated to use `mod()`